src/promptflow/promptflow/_telemetry/activity.py

```diff
@@ -92,6 +96,37 @@ def log_activity(
             raise exception
 
 
+def add_telemetry_log(
+    activity_name,
+    activity_type=ActivityType.INTERNALCALL,
+    message='',
+    custom_dimensions=None,
+    level='info'
+):
+    from promptflow._telemetry.telemetry import get_telemetry_logger
+
+    logger = get_telemetry_logger()
+    custom_dimensions = custom_dimensions or {}
+    context = OperationContext.get_instance()
+    request_id = custom_dimensions.get('request_id', context.get('request_id', str(uuid.uuid4())))
```

Comment:
Does operation context have `request_id`?

Reply:
I forgot to modify this. Initially, I recorded the request id, but later recorded the entire **last_log_activity_info** information, request id in **last_log_activity_info**